scripting/smrpg_keyhint_info.sp

```diff
@@ -47,6 +53,12 @@ public void OnPluginStart()
 	AutoExecConfig_SetPlugin(null);
 
 	g_hCVDefaultHidePanel = AutoExecConfig_CreateConVar("smrpg_hide_infopanel_default", "0", "Hide the info panel by default for new players? They'll have to enable it themselves.", 0, true, 0.0, true, 1.0);
+	gc_fX = AutoExecConfig_CreateConVar("sm_hud_x", "-1", "x coordinate, from 0 to 1. -1.0 is the center of sm_hud_type '1'", _, true, -1.0, true, 1.0);
+	gc_fY = AutoExecConfig_CreateConVar("sm_hud_y", "0.1", "y coordinate, from 0 to 1. -1.0 is the center of sm_hud_type '1'", _, true, -1.0, true, 1.0);
+	gc_iRed = AutoExecConfig_CreateConVar("sm_hud_red", "0", "Color of sm_hud_type '1' (set R, G and B values to 255 to disable) (Rgb): x - red value", _, true, 0.0, true, 255.0);
```

Comment:
Is the reference to `sm_hud_type` from some older changes you made?
The convars should be prefixed with `smrpg_` instead of `sm_` by the way.